input/pagecontent/changes.md

```diff
@@ -8,6 +8,7 @@ This change log documents the significant updates and resolutions implemented fr
 #### Changes in this version
 
 - Added Security and Privacy page [FHIR-45067](https://jira.hl7.org/browse/FHIR-45067).
+- Added Medicine Information page [FHIR-45165](https://jira.hl7.org/browse/FHIR-45165).
```

Comment:
quality this to be clear the information from general guidance on medicine information was moved to it's own page.

Reply:
Fixed.